_posts/2020-06-07-Human-Curation-and-Digital-Datasets.md

```diff
@@ -184,16 +282,17 @@ may be visualised thus:
 However, due to the structure of the data, errors can be detected more
 easily if the visualisation does not make sense.  Many times this can
 be identified as a cyclical structure in the descendants of an
-individual where one of the descendants of an individual ends up being
-both an indirect and direct descendant of an individual who should be
-farther up the genealogical structure.  This visual inspection is the
-basis for Thansich's previous article and for his current work on
-IrishGen.
+individual where one of the descendants ends up being both an indirect
+and direct descendant of an individual.  In other words, a descendant
+several generations away from their ancestor is also encoded as their
+direct ancestor which creates a cyclical structure in the genealogy.
+This visual inspection is the basis for Thansich's previous article
```

Comment:
Perhaps note that my previous article doesn't include examples of the Phillip J. Fry syndrome in action, just examples of use of the visualiser.  There will surely be a blogpost about the Phillip J. Fry syndrome in the the fairly near future so perhaps note that this will be covered.